perfer.config.js

```diff
@@ -1,11 +1,18 @@
 const files = [
   ['ActionsMenuLayout.bundle.min.js', 6],
+  ['ActionsMenuLayoutPerfBasic.bundle.min.js', 6],
+  ['ActionsMenuLayoutPerfExtended.bundle.min.js', 6],
   ['AddItem.bundle.min.js', 45.5],
   ['Autocomplete.bundle.min.js', 37],
   ['Avatar.bundle.min.js', 16],
   ['AvatarGroup.bundle.min.js', 19],
   ['Badge.bundle.min.js', 5],
+  ['BadgePerfBasic.bundle.min.js', 5],
+  ['BadgePerfExtendedBad.bundle.min.js', 5],
+  ['BadgePerfExtendedGood.bundle.min.js', 5],
   ['Button.bundle.min.js', 9],
+  ['ButtonPerfBasic.bundle.min.js', 9],
+  ['ButtonPerfExtended.bundle.min.js', 10],
```

Comment:
@deanshub 
You can see for instance that I've added two bundles:
1. is a basic component using our component
2. the same basic component that in addition overrides the default stylings (using stylable)

This will give us a relatively accurate indication on how overriding styles affects the bundle.
I'm not sure about the naming, I wanted it to be something like: `Button.perf.basic.bundle.min.js` and `Button.perf.extended.bundle.min.js`, but this creates a name collision for stylable's compiler.

Reply:
looks good from my standpoint as is